README.md

```diff
@@ -195,6 +193,12 @@ if you accidentally leave them in code, however (same for `undefined`).
 
 We also have `data Undefined = Undefined` (which, too, comes with warnings).
 
+### Exceptions
+
+We use [`safe-exceptions`](https://github.com/fpco/safe-exceptions)
+library for exceptions handling. Don't export `Control.Exceptions`
```

Comment:
Do you mean "don't import"? I believe you could force this by exporting `Control.Exception` as well, causing a conflict (it'll be possible to resolve the conflict using `-XPackageImports`, but then you'll know it's intentional).

Reply:
Yes, you're right. I want to say _don't import_.